src/Compilers/CSharp/Portable/Symbols/AnonymousTypes/SynthesizedSymbols/AnonymousType.DelegateTemplateSymbol.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.CSharp.Emit;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    internal sealed partial class AnonymousTypeManager
+    {
+        internal sealed class AnonymousDelegateTemplateSymbol : AnonymousTypeOrDelegateTemplateSymbol
+        {
+            private readonly ImmutableArray<Symbol> _members;
+
+            /// <summary>
+            /// True if the parameter types and return type of the delegate are all type parameters.
+            /// </summary>
+            internal readonly bool IsParameterizedDelegateType;
```

Comment:
>IsParameterizedDelegateType [](http://example.com/codeflow?start=35&length=27)

I think the usage of "Parameterized" term is confusing. None of the assumptions that come to mind based on this term actually match what we are trying to convey here.